### DISCUSSION TITLE
BackEndBr - Fórum

### DISCUSSION THREAD

**Discussion by onhernandes** [frontendbr/forum#666] [Dicas]
Não sei se vocês viram, mas agora a gente tem um [fórum pra backendbr](https://github.com/backend-br/forum/issues) também, igualzinho esse aqui. O projeto todo foi baseado nesse frontendbr, só que pro backend. Recomendo todo mundo dar um pulo lá e dar umas stars nos repos e fazer perguntas sobre backend por lá também!

**Comment by WandersonAlves** [comment]
Tinha que ser a issue #666 mesmo HRIRORORIEEYYW

Brinks, boa iniciativa!

**Comment by onhernandes** [comment]
@WandersonAlves 
AHUAHAUAHAUUAHAUAHUA WOOOW

É noix

**Comment by bernardodiasc** [comment]
HUAHAUHAUAHUA \m/ 666 \m/

**Comment by felipefialho** [comment]
Da hora!
